build.gradle

```diff
@@ -173,7 +173,7 @@ project(':ui') {
     script = file('node_modules/.bin/webpack')
   }
 
-  tasks.build.dependsOn(lint)
+  // tasks.build.dependsOn(lint)
```

Comment:
Should we delete this line?

Reply:
We can; I preferred just commenting it in case one day we want to re-enable it. 
TBH I don't expect any of the team member to start writing nodejs and fixing "just" linting errors for this deprecated project :/

Reply:
Ack, thanks :)